setup/Swix/Microsoft.FSharp.Compiler.MSBuild/Microsoft.FSharp.Compiler.MSBuild.csproj

```diff
@@ -84,31 +84,31 @@ vs.dependencies
 folder "InstallDir:Common7\IDE\CommonExtensions\Microsoft\FSharp"
   file source="$(BinariesFolder)\fsc\$(Configuration)\$(TargetFramework)\fsc.exe" vs.file.ngen=yes vs.file.ngenArchitecture=All vs.file.ngenPriority=2 vs.file.ngenApplication="[installDir]\Common7\IDE\CommonExtensions\Microsoft\FSharp\fsc.exe"
   file source="$(BinariesFolder)\fsc\$(Configuration)\$(TargetFramework)\fsc.exe.config"
+  file source="$(BinariesFolder)\fsc\$(Configuration)\$(TargetFramework)\Microsoft.Build.dll"
```

Comment:
I'd rather these references come specifically from the project output dir that originally requested them, e.g., FSharp.Compiler.Private for most.

Reply:
yes I see